gap/weights.gi

```diff
@@ -772,6 +772,54 @@ function(D, start, destination)
   return flows;
 end);
 
+InstallMethod(DigraphMinimumCutSet, "for an edge weighted digraph",
+[IsDigraph and HasEdgeWeights, IsPosInt, IsPosInt],
+function(D, s, t)
+  local weights, outs, vertices, flow, residuals, G, o, u, v, S, T;
+
+  # Extract important data
+  weights := EdgeWeights(D);
+  outs := OutNeighbours(D);
+  vertices := DigraphVertices(D);
+
+  # Check input
+  if not s in vertices then
```

Comment:
Minor optimization: Because digraphs must be homogenous (all vertices are in the range `[1 .. DigraphNrVertices(D)]` with no holes), it might be better performance to check if `s >= 1` and `s <= DigraphNrVertices(D)`.